src/helpers/AIAssistant/Tools.ts

```diff
@@ -262,3 +262,59 @@ export const GetEhrNewsFeedPageTool = tool(
     })
   }
 );
+
+export const GraphingTool = tool(
+  async (input): Promise<string> => {
+
+    let response = await fetch('https://kwejahcwmsyzidk5vde5uyv33a0saruz.lambda-url.us-east-1.on.aws', {
+      method: 'POST',
+      body: JSON.stringify({ code: input.code }),
+      headers: { 'Content-Type': 'application/json' }
+    }).then(response => response.json());
+
+    return response.image;
```

Comment:
This should always return a string. If something goes wrong with the response, an invalid content value can get stuck in the OpenAI message history. Subsequent requests are then rejected by OpenAI.

Maybe:
```
return response.image || "";
//or get crazy?
return response.image || "An unexpected error occurred while executing the tool; please try again.";
```